misc/images/tidb-debug/Dockerfile

```diff
@@ -27,8 +27,8 @@ RUN yum update -y && yum install -y \
 
 RUN wget -q http://download.pingcap.org/tidb-latest-linux-amd64.tar.gz \
     && tar xzf tidb-latest-linux-amd64.tar.gz \
-    && mv tidb-latest-linux-amd64/bin/* /usr/local/bin/ \
```

Comment:
What's this dockerfile used for?
The definition of `latest` has changed to the latest stable release and `nightly` now is the tag for the original latest daily build.
Do we need to change to nightly here?

Reply:
cc @aylei

Reply:
this is used for debug running tidb clusters in Kubernetes, it's safe to use the stable release.

Reply:
Yes